### DISCUSSION TITLE
[RFC Call] Agenda - February 8, 2021

### DISCUSSION THREAD

**Discussion by jonathantneal** [withastro/roadmap#98] [Proposal]
# Community RFC Call

:astronaut: Our **Community RFC Call** begins <a href="https://hammertime.djdavid98.art/?t=1644346800"><time datetime="2022-08-01T19:00:00.000Z">Tue, 08 Feb 2022 19:00 GMT<time></a>  :astronaut:

- [Join](https://discord.gg/22XmBt49)
- [Chat](https://canary.discord.com/channels/830184174198718474/940671590289510440/940671593644982362)

## Agenda

- Anonymous telemetry, with @natemoo-re
- Assets, with @FredKSchott

Time permitting, we’ll go thru other callouts in the comments.

---

This event is open to everyone in the community. Join us to discuss proposals that will improve and enhance Astro.

If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests](https://github.com/withastro/rfcs/discussions).

**Comment by zadeviggers** [comment]
I would like it if #102 could be discussed. Unfortunately due to real life I won't be able to attend the RFC call myself, but I'd still love if it could be discussed nonetheless.

**Reply by jonathantneal** [reply]
Thanks for the suggestion and heads up, @zadeviggers. If time allows, @matthewp, seeing your comment, would you be interested in discussing this?

**Comment by FredKSchott** [comment]
I have been doing a ton of research on assets, and I know a lot of our community has been looking at assets (images, icons, etc) as well. I would love to take some time to present what I've found!

**Reply by jonathantneal** [reply]
I’ll do my best to get it captured in notes, but if possible, would you be able to follow this with a discussion post, gist, or link?

**Comment by natemoo-re** [comment]
I would like to discuss to possibility of adding **anonymous** telemetry to the Astro CLI. I have been putting a lot of thought into this, so would love to discuss what kind of data we'd be collecting and why it would be useful for us.

**Reply by jonathantneal** [reply]
I’ll do my best to get it captured in notes, but if possible, would you be able to follow this with a discussion post, gist, or link?

**Comment by jonathantneal** [comment]
Time permitting, I would like to make a brief request for eyes on [#105: Attribute Resolution](https://github.com/withastro/rfcs/discussions/105).
